### PR TITLE
fix(anta.cli): Fix minor glitches and cleanup

### PR DESCRIPTION
**Pull request by mtache**
- fix(anta.cli) refactor collect_commands(). Fix output_directory input var
- fix(anta.cli) Remove no_args_is_help for 'anta nrfu table' and 'anta nrfu json' otherwise env var are not considered
- fix(anta.cli): remove default value of inventory, check if file exists and is readable
- clean(scripts): remove unused scripts that are now implemented as a CLI entrypoint
- clean(anta.inventory): Remove paramiko library

### CODE REVIEW

File: anta/cli/cli.py [hunk 1]
```diff
@@ -19,12 +19,12 @@
 @click.group()
 @click.pass_context
 @click.version_option(__version__)
-@click.option('--username', show_envvar=True, default='arista', help='Username to connect to EOS', required=True)
+@click.option('--username', show_envvar=True, default='admin', help='Username to connect to EOS', required=True)
```
Comment (gmuloc) [inline anta/cli/cli.py:22]: so I don't really care about the default value but does it change the outputs of ythe help and so the documentation?

File: anta/cli/exec/commands.py [hunk 2]
```diff
@@ -43,12 +44,17 @@ def clear_counters(ctx: click.Context, log_level: str, tags: str) -> None:
...
+
+
 @click.command()
 @click.pass_context
 # Generic options
 @click.option('--tags', '-t', default=DEFAULT_TAG, help='List of tags using coma as separator: tag1,tag2,tag3', type=str, required=False)
 @click.option('--commands-list', '-c', show_envvar=True, type=click.Path(), help='File with list of commands to grab', required=True)
-@click.option('--output-directory', '-outut', '-o', show_envvar=True, type=click.Path(), help='Path where to save commands output', required=False)
+@click.option('--output-directory', '-output', '-o', show_envvar=True, type=click.Path(), help='Path where to save commands output',
+              default='anta_snapshot', callback=_get_snapshot_dir, required=True)
```
Comment (gmuloc) [inline anta/cli/exec/commands.py:57]: required True + default?

File: anta/cli/exec/commands.py [hunk 3]
```diff
@@ -43,12 +44,17 @@ def clear_counters(ctx: click.Context, log_level: str, tags: str) -> None:
         )
 
 
+def _get_snapshot_dir(ctx: click.Context, param: click.Parameter, value: str) -> str:  # pylint: disable=unused-argument
```
Comment (titom73) [inline anta/cli/exec/commands.py:47]: Maybe just a basic docstring to help understand the goal of it

File: anta/cli/check/commands.py [hunk 4]
```diff
@@ -62,7 +62,7 @@ def table(ctx: click.Context, catalog: str, tags: str, group_by: str, search: st
     return True
 
 
-@click.command(no_args_is_help=True)
```
[the remark this answers is not in the document]
Comment (titom73) [reply anta/cli/check/commands.py:65]: Ok :)

File: anta/cli/cli.py (hunk 1 above)
Comment (mtache) [reply anta/cli/cli.py:22]: No, nothing in doc.

File: anta/cli/exec/commands.py (hunk 2 above)
Comment (mtache) [reply anta/cli/exec/commands.py]: > Require #166 to be merged first

Not anymore

File: anta/cli/exec/commands.py (hunk 3 above)
Comment (mtache) [reply anta/cli/exec/commands.py]: Not in this PR anymore, see #166

File: anta/cli/exec/commands.py (hunk 2 above)
Comment (mtache) [reply anta/cli/exec/commands.py]: Not in this PR anymore, see #166

**Review by gmuloc — APPROVED** [review]

**Comment by gmuloc** [comment]
waiting for #168 to merge

**Pull request merged**